packages/aws-cdk/lib/api/deploy-stack.ts

```diff
@@ -151,7 +151,16 @@ export async function deployStack(options: DeployStackOptions): Promise<DeploySt
     // tslint:disable-next-line:max-line-length
     const monitor = options.quiet ? undefined : new StackActivityMonitor(cfn, deployName, options.stack, (changeSetDescription.Changes || []).length).start();
     debug('Execution of changeset %s on stack %s has started; waiting for the update to complete...', changeSetName, deployName);
-    await waitForStack(cfn, deployName);
+    try {
+      await waitForStack(cfn, deployName);
+    } catch (e) {
```

Comment:
why not `finally`?

Reply:
no good reason, that's cleaner! thanks for the suggestion